packages/mobile/src/send/reducers.ts

```diff
@@ -42,8 +40,22 @@ export const sendReducer = (
       }
     }
     case Actions.SEND_PAYMENT_OR_INVITE:
-      return sendPaymentOrInvite(state, action.amount, action.timestamp)
+      return {
+        ...state,
+        isSending: true,
+      }
     case Actions.SEND_PAYMENT_OR_INVITE_SUCCESS:
+      const now = Date.now()
+      // Keep only the last 24 hours
+      const paymentsLast24Hours = state.recentPayments.filter(
+        (p: PaymentInfo) => timeDeltaInHours(now, p.timestamp) < 24
+      )
+      const latestPayment: PaymentInfo = { amount: action.amount.toNumber(), timestamp: now }
+      return {
```

Comment:
Do you need to set `isSending` to false here?

Reply:
I would think so but that's not how it was setup before. Shouldn't hurt to add that in so I'll go ahead and do that.